pkg/registry/manifest.go

```diff
@@ -23,6 +23,9 @@ import (
 	"net/http"
 	"strings"
 	"sync"
+
+	v1 "github.com/google/go-containerregistry/pkg/v1"
```

Comment:
I'd prefer it if we inlined some of this stuff to avoid taking a dependency outside of the stdlib for this package, but I might just fix it myself before we cut a new release.